core/src/main/java/org/ehcache/core/spi/cache/Store.java

```diff
@@ -208,9 +255,11 @@
   StoreEventSource<K, V> getStoreEventSource();
 
   /**
-   * Returns an iterator over the elements in this store.  The elements are
-   * returned in no particular order (unless this set is an instance of some
+   * Returns an iterator over the elements in this store.
+   * <P>
+   *   The elements are returned in no particular order (unless this set is an instance of some
```

Comment:
Whats the point of the "unless..." clause?  "Instances of this interface behave as document unless documented to behave differently"?